{{ cookiecutter.project_slug }}/jest.config.js

```diff
@@ -8,4 +8,10 @@ module.exports = {
     "^.+\\.ts$": "ts-jest",
   },
   verbose: true,
+  // as per https://github.com/codecov/example-typescript/blob/main/package.json:
+  collectCoverage: true,
+  coverageReporters: [
+    "text",
```

Comment:
thanks to this every local test run also outputs something like below on the console (even colored)
```
----------|---------|----------|---------|---------|-------------------
File      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
----------|---------|----------|---------|---------|-------------------
All files |     100 |      100 |     100 |     100 |                   
 lib.ts   |     100 |      100 |     100 |     100 |                   
----------|---------|----------|---------|---------|-------------------
```